clients/drcachesim/tests/record_filter_unit_tests.cpp

```diff
@@ -170,6 +172,17 @@ struct test_case_t {
     std::vector<bool> output;
 };
 
+struct test_entries_t {
+    /* Represents entry in the input trace.
+     */
+    trace_entry_t entry_test;
+    /* Represents corresponding expected entry in the output trace.
+     * XXX i#6662: consider making this into a std::vector<trace_entry_t> to test cases
+     * where DR_ISA_REGDEPS encoding is longer or shorter than entry_test.
+     */
+    trace_entry_t entry_expected;
```

Comment:
But the tests are all using test_case_t.  Best to use the same instead of making a similar new scheme.  I don't see anything here that doesn't fit test_case_t.

Reply:
I misunderstood how test_case_t works. Fixed using existing infrastructure.